executor/infoschema_reader.go

```diff
@@ -2174,26 +2180,100 @@ func dataForAnalyzeStatusHelper(sctx sessionctx.Context) (rows [][]types.Datum,
 		if !chunkRow.IsNull(10) {
 			procID = chunkRow.GetUint64(10)
 		}
-		rows = append(rows, types.MakeDatums(
-			dbName,        // TABLE_SCHEMA
-			tableName,     // TABLE_NAME
-			partitionName, // PARTITION_NAME
-			jobInfo,       // JOB_INFO
-			processedRows, // ROW_COUNT
-			startTime,     // START_TIME
-			endTime,       // END_TIME
-			state,         // STATE
-			failReason,    // FAIL_REASON
-			instance,      // INSTANCE
-			procID,        // PROCESS_ID
-		))
+
+		var remainDurationStr, progressStr, estimatedRowCntStr interface{}
+		if isShow && state == statistics.AnalyzeRunning {
+			RemainingDuration, progress, estimatedRowCnt, RemainDurationErr :=
+				getRemainDurationForAnalyzeStatusHelper(sctx, startTime,
+					dbName, tableName, partitionName, processedRows)
+			if RemainDurationErr != nil {
+				log.Warn("get remaining duration failed", zap.Error(RemainDurationErr))
```

Comment:
Do we need to use `logutil.BgLogger().Warn` here?

Reply:
Done